controllers/operatorpolicy_status.go

```diff
@@ -578,6 +594,82 @@ func catalogSrcUnknownObj(catalogName string, catalogNS string) policyv1.Related
 	}
 }
 
+// noInstallPlansCond is a Compliant condition with Reason 'NoInstallPlansFound',
+// and Message 'there are no relevant InstallPlans in the namespace'
+var noInstallPlansCond = metav1.Condition{
+	Type:    installPlanConditionType,
+	Status:  metav1.ConditionTrue,
+	Reason:  "NoInstallPlansFound",
+	Message: "there are no relevant InstallPlans in the namespace",
+}
+
+// installPlanFailed is a NonCompliant condition with Reason 'InstallPlanFailed'
+// and message 'the current InstallPlan has failed'
+var installPlanFailed = metav1.Condition{
+	Type:    installPlanConditionType,
+	Status:  metav1.ConditionFalse,
+	Reason:  "InstallPlanFailed",
+	Message: "the current InstallPlan has failed",
+}
+
+// installPlanInstallingCond is a NonCompliant condition with Reason 'InstallPlansInstalling'
+// and message 'a relevant InstallPlan is actively installing'
+var installPlanInstallingCond = metav1.Condition{
+	Type:    installPlanConditionType,
+	Status:  metav1.ConditionFalse,
+	Reason:  "InstallPlansInstalling",
+	Message: "a relevant InstallPlan is actively installing",
+}
+
+// installPlansFineCond is a Compliant condition with Reason 'InstallPlansFine'
```

Comment:
"Fine" is too abstract/non-specific and I feel like it wouldn't be widely understood when translated. Unless this is some alignment with OLM, is there a better word like "Succeeded", "Complete", or "Installed"?

Reply:
I agree, I had no better ideas when I made it up 😆 

I think originally I thought this would need to cover more cases than it does - so I deliberately avoided something like "Succeeded" because I expected it would cover a case where nothing had been installed yet... but now I think this situation is really just "there are no InstallPlans considered for approval", so I can report that as the condition.